### ISSUE TITLE
VoiceOver can't read site assembly screen

### ISSUE THREAD

**Issue by guarani** [wordpress-mobile/WordPress-iOS#22189]
### Expected behavior

After creating a new site, the screen shown (aka. the site assembly screen) should be fully accessible by VoiceOver.

### Actual behavior

VoiceOver can't read many elements on this screen, such as the "We've emailed you your receipt." and "It may take up to 30 minutes..." messages.

### Steps to reproduce the behavior
1. Create a new site
2. Stop on the screen that says "Kudos, your site is live!"
3. Notice that VoiceOver doesn't read parts of the screen

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
